src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -78,7 +79,15 @@ export function UnfillableOrdersUpdater(): null {
       const isUnfillable = isOrderUnfillable(order, price)
 
       // Only trigger state update if flag changed
-      order.isUnfillable !== isUnfillable && setIsOrderUnfillable({ chainId, id: order.id, isUnfillable })
+      if (order.isUnfillable !== isUnfillable) {
```

Comment:
this is strange, they're the same thing as isOrderUnfillable takes the order and derives info from it. why not just add isUnfullable in the dependency array

Reply:
As far as I can see I don't have order and price outside of that useCallback since they are passed as parameters. So I can't calculate isUnfillable outside of useCallback